src/test/regress/pg_regress_multi.pl

```diff
@@ -462,7 +462,6 @@ END
 # Finally run the tests
 if ($vanillatest)
 {
-    push(@arguments, "--schedule=${postgresSrcdir}/parallel_schedule");
```

Comment:
I'm guessing the intention here was to directly start pg_regress below, which would allow us to add `--load-extension=citus`.